Makefile

```diff
@@ -105,3 +105,14 @@ delete-traces:
 	kubectl delete -k ./cluster-infra/otel-operator/ --ignore-not-found
 	kubectl delete -k ./cluster-infra/cert-manager/ --ignore-not-found
 	kubectl delete -f https://github.com/flux-iac/tofu-controller/releases/download/v0.15.1/tf-controller.crds.yaml
+
+.PHONY: tag-o11y-restart
+tag-o11y-restart:
```

Comment:
nitpick:
I feel like we could simplify this to `restart` since it doesn't conflict with other targets

Reply:
good point, I can just have the target be called `restart`